### DISCUSSION TITLE
Redirecting YouTube issue creators to the current issue

### DISCUSSION THREAD

**Discussion by ghost** [uBlockOrigin/uBlock-issues#3009] [Ideas]
Perhaps a verification could be implemented that results in the redirection of the user to the proper issue.

And while we're thinking about this, what about a verification for filter update time (<20 minutes => issue unable to be filed uBO)

**Comment by garry-ut99** [comment]
> Omega-6: Redirecting YouTube issue creators to the current issue
> Omega-6: Perhaps a verification could be implemented that results in the redirection of the user to the proper issue.

You are 4th person proposing the same: https://github.com/uBlockOrigin/uAssets/issues/19976#issuecomment-1762689019
